packages/protocol/contracts-0.8/common/EpochManager.sol

```diff
@@ -319,4 +336,42 @@ contract EpochManager is
       CELOequivalent
     );
   }
+
+  /**
+   * @notice Sends the allocated epoch payment to a validator, their group, and
+   *   delegation beneficiary.
+   * @param validator Account of the validator.
+   */
+  function sendValidatorPayment(address validator) external {
+    IAccounts accounts = IAccounts(getAccounts());
+    address signer = accounts.getValidatorSigner(validator);
```

Comment:
I believe we didn't want to use signers @mcortesi @martinvol

Reply:
I was basing this on how the current code works:
* the `elected` array in EpochManager gets set by calling `Election.electNValidatorSigners`
* That functions gets validator addresses via `Validators.getTopGroupValidators`
* Which grabs the validator accounts and translates them to validator signer addresses, returning those

So in the current state, the correct way to index into `elected` is via signer addresses.

If we want to change this logic, we should thoroughly double check everything is returning/using the correct type of validator address. As mentioned on Slack, a similar issue exists with the current use of `Validators.computeEpochReward`, which expects a validator account, but gets a signer address (generated by the election). I would suggest this be done in a follow-up issue/PR that covers all instances of this.

In general I agree, we can probably just use account addresses everywhere. If I understand correctly, the only reason we used signer addresses in some places was because that's what celo-blockchain cared about in the PoS logic.

Reply:
yes, we don't want signeers. EpochManager will have a list of elected account (not signers) and we should change elections to not give signers, i believe we even have a draft methos doing that...

the client didn't care about account, only about signers and so it sent signers everywhere, but that's unnecessary complexity now

Reply:
Created an [issue](https://github.com/celo-org/celo-blockchain-planning/issues/589) to track progress on this

Reply:
@mcortesi, sounds great, that's how I understood things.

As mentioned, I would merge this PR as is, so it's consistent with the current state of the logic, and remove signers in the PR that will fix @soloseng's issue.

Reply:
[PR](https://github.com/celo-org/celo-monorepo/pull/11201) making the changes.